x-pack/plugins/actions/server/plugin.test.ts

```diff
@@ -737,6 +737,12 @@ describe('Actions Plugin', () => {
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
           const pluginSetup = await plugin.setup(coreSetup as any, pluginsSetup);
 
+          const platinumLicense = licensingMock.createLicense({
```

Comment:
I think the test was using "basic" here, so the new code path caused this test to fail.  Upgrading the license to a cases-friendly license to get the test working again.